__tests__/app.test.js

```diff
@@ -33,8 +34,35 @@ describe('/api/categories', () => {
     return request(app)
       .get('/api/notAPath')
       .expect(404)
-      .then(({body}) => {
-        expect(body.msg).toBe('Invalid Path');
-      });  
+      .then(({ body }) => {
+        expect(body.msg).toBe('Path not found');
+      });
+  });
+});
+
+describe('/api/reviews', () => {
+  it('200: GET responds with an array of review objects sorted by descending date', () => {
+    return request(app)
+      .get('/api/reviews')
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeInstanceOf(Array);
+        expect(reviews).toHaveLength(13);
+        expect(reviews).toBeSortedBy('created_at', { descending: true });
```

Comment:
I really like the use of jestSorted here to assert the results are in the correct order!